packages/nx/src/command-line/login/command-object.ts

```diff
@@ -0,0 +1,20 @@
+import { CommandModule } from 'yargs';
+import { withVerbose } from '../../command-line/yargs-utils/shared-options';
+
+export const yargsLoginCommand: CommandModule = {
+  command: 'login [nxCloudUrl]',
+  describe:
+    'Login to Nx Cloud by generating a personal access token and saving it to your local machine.',
```

Comment:
This seems like too much information about the implementation details.

I think we can shorten it to..
```suggestion
    'Login to Nx Cloud',
```